tutor/commands/upgrade/common.py

```diff
@@ -43,6 +43,39 @@ def upgrade_from_nutmeg(context: click.Context, config: Config) -> None:
     )
 
 
+def get_mysql_change_authentication_plugin_query(config: Config) -> str:
+    """Helper function to generate queries depending on the loaded plugins"""
```

Comment:
the docstring can be upgraded to highlight what is done by default and what is done based on plugin's availability.

Reply:
Updated.